OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+to be normalized and made comparable to the NPX values of an Olink
+Explore HT project. This process is one-directional, and normalizing
+Olink Explore HT NPX values to Olink Explore 3072 is not supported.
+
+The product bridging normalization uses the **\~**2900 assays that are
+overlapping between Olink Explore 3072 and Olink Explore HT. Each
+overlapping assay undergoes a series of checks that evaluate the number
+of counts, correlation, and difference of NPX ranges between the two
+data sets. If an assay has enough counts and comparable metrics between
+the two data sets, it is determined to be suitable for bridging
+(referred to as a "bridgeable assay"). Assays that are not suitable for bridging can either be excluded from downstream analysis in one or both products or results can be integrated across products using meta-analysis. The set of bridgeable assays
+across products will vary from data set to data set, based on the
+samples present within the studies. Depending on the NPX distribution of
+each bridgeable assay in the two data sets, the assay is normalized
+using either median normalization or quantile smoothing.
+
+Bridging an Explore 3072 data set to an Explore HT NPX data set
+requires 40 - 64 bridging samples. Bridging samples are shared samples
+among data sets and, as such, are analyzed in both data sets. Olink NPX
+data sets without shared samples cannot be combined using the
+bridging approach described below. More information on bridge sample
+selection can be found in the [selecting bridging samples
+section](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/bridging_introduction.html#selecting-bridging-samples)
+of the Introduction to Bridging tutorial.
+
+## Bridge Sample Selection
+
+Prior to running a study with Explore HT, bridging samples must be selected
+from the study run with Explore 3072 and be run on the Explore HT study. These samples can
+be selected using the `olink_bridgeselector()` function in Olink Analyze
+as detailed in [the Introduction to bridging
+tutorial](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/bridging_introduction.html#selecting-bridging-samples). The recommended number of bridge samples for within- and between- product bridging is summarized in the table below. When selecting bridge samples, the aim is to select samples that represent the dynamic range of the assay expression in the product. As such, quality control of the sample and, if available, proportion of data above LOD in the sample are considered when determining if a sample is chosen as a bridging sample. When LOD data is not available in the data expert from Olink NPX software, LOD can optionally be calculated from fixed LOD or negative controls as
+detailed in the [Calculating LOD from Olink Explore data
+tutorial](https://cran.r-project.org/web/packages/OlinkAnalyze/vignettes/LOD.html).
+
+```{r brnrtab, eval=TRUE, message=FALSE}
```

Comment:
This script section is printing in the vignette. Add "echo = F" to print just the table and not the corresponding script.